ic/terabethia/src/tera/src/api/consume_message.rs

```diff
@@ -44,8 +64,8 @@ fn consume(from: Principal, payload: Vec<Nat>) -> Result<bool, String> {
     if res.is_ok() {
         let store = STATE.with(|s| s.store_outgoing_message(msg_hash, !MESSAGE_PRODUCED));
         match store {
-            Err(e) => panic!("{:?}", e),
-            _ => (),
+            Ok(_) => STATE.with(|s| s.update_nonce(nonce)),
+            Err(error) => panic!("{:?}", error),
         }
     }
```

Comment:
this is no longer needed, because we're not updating L1 state of consumed messages (L1 -> L2).